internal/config/config.go

```diff
@@ -240,29 +251,45 @@ func (c *AuthConfig) Login(hostname, username, token, gitProtocol string, secure
 	var setErr error
 	if secureStorage {
 		if setErr = keyring.Set(keyringServiceName(hostname), "", token); setErr == nil {
-			// Clean up the previous oauth_token from the config file.
+			setErr = keyring.Set(keyringServiceName(hostname), username, token)
```

Comment:
Would appreciate this being commented to indicate why we're doing both the blank username and provided username.